cmd/gke-gcloud-auth-plugin/cred/cred.go

```diff
@@ -97,20 +111,29 @@ type Options struct {
 func PrintCred(options *Options) error {
 	if options == nil {
```

Comment:
Looking at this, it no longer makes sense to have the main.go and cred dir abstraction. Can we remove merge the main.go, rename cred.go to main.go, move all files one level up and just rename PrintCred to main()?